deploy/operator/base/deployment.yaml

```diff
@@ -25,6 +25,10 @@ spec:
           args:
             - "--v=4"
             - "--zap-log-level=info"
+          env:
+            - name: S3_SYNC_ENABLED
+              value: |
+                ${JBS_S3_SYNC_ENABLED}
```

Comment:
This won't work with argocd, so it will break the infra-deployments repo.

Reply:
You mean the infra stuff processes that file directly (somehow?) and will see a unknown string instead of it being substituted? As opposed to argocd being unable to process certain yaml formats?

Reply:
Infra deployments does not use our development or CI scripts, it is using Argo CD, so it just doesn't know about the substitution.

Reply:
Ok, thats fine. I've removed the env variable